js/css/feedback.scss

```diff
@@ -0,0 +1,53 @@
+.pop {
+    background-color: #262625;
```

Comment:
You shouldn't hard-code colors because the user can change the book's theme. Instead, I would find a set of CSS theme variables that look reasonable. See: https://github.com/rust-lang/mdBook/blob/master/src/theme/css/variables.css

Reply:
Added theme colors - omitted the `.no-js` selector since the tooltip won't show up if JS is disabled.